esp/src/eclwatch/dojoConfig.js

```diff
@@ -32,8 +32,10 @@ function getConfig(env) {
             "@hpcc-js/map": baseUrl + "/node_modules/@hpcc-js/map/dist/map",
             "@hpcc-js/other": baseUrl + "/node_modules/@hpcc-js/other/dist/other",
             "clipboard": baseUrl + "/node_modules/clipboard/dist/clipboard",
+            "codemirror": baseUrl + "/node_modules/codemirror",
+            "crossfilter": baseUrl + "/crossfilter/crossfilter.min",
             "font-awesome": baseUrl + "/node_modules/@hpcc-js/common/font-awesome",
-            "crossfilter": baseUrl + "/crossfilter/crossfilter.min"
+            "tslib": baseUrl + "/node_modules/tslib/tslib"
```

Comment:
This code mirror related?

Reply:
tslib is needed by CodeMirror.ts specifically its a couple of helper functions for TypeScript classes and its getting added now, as its the first TS class we are adding (there will be more)..